client/src/components/layout/AccountPage.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from "react";
+import _ from "lodash";
+import Dropzone from "react-dropzone";
+
+const AccountPage = (props) => {
+  const [user, setUser] = useState(props.user);
+  const handleImageUpload = (acceptedImage) => {
+    setUser({ ...user, image: acceptedImage[0] });
```

Comment:
`user` state is the "source of truth" for this component of the info it is displaying. The form to upload an image needs to be set up as a controlled component where it has its own state, separate from the data displayed on the page. This form state is the "temporary state" that the user is interacting with before they hit submit to add the image